Warning about implicit fallthrough since Linux 5.14

This warning is back, despite the fallthrough comment added in #12:

```
In file included from /home/fedora/actions-runner/_work/broadcom-wl/broadcom-wl/src/wl/sys/wl_cfg80211_hybrid.c:43:
/home/fedora/actions-runner/_work/broadcom-wl/broadcom-wl/src/wl/sys/wl_cfg80211_hybrid.c: In function ‘wl_set_auth_type’:
/home/fedora/actions-runner/_work/broadcom-wl/broadcom-wl/src/wl/sys/wl_cfg80211_hybrid.h:57:12: warning: this statement may fall through [-Wimplicit-fallthrough=]
   57 |         if (wl_dbg_level & WL_DBG_DBG) {                        \
      |            ^
/home/fedora/actions-runner/_work/broadcom-wl/broadcom-wl/src/wl/sys/wl_cfg80211_hybrid.c:826:17: note: in expansion of macro ‘WL_DBG’
  826 |                 WL_DBG(("network eap\n"));
      |                 ^~~~~~
/home/fedora/actions-runner/_work/broadcom-wl/broadcom-wl/src/wl/sys/wl_cfg80211_hybrid.c:828:9: note: here
  828 |         default:
      |         ^~~~~~~
```

https://gcc.gnu.org/onlinedocs/gcc-12.1.0/gcc/Warning-Options.html#index-Wimplicit-fallthrough

For kernel versions ≥ 5.4, we should probably switch to the `fallthrough` pseudo-keyword:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=294f69e662d1570703e9b56e95be37a9fd3afba5

--- a/src/wl/sys/wl_cfg80211_hybrid.c
+++ b/src/wl/sys/wl_cfg80211_hybrid.c
@@ -824,7 +824,9 @@ wl_set_auth_type(struct net_device *dev, struct cfg80211_connect_params *sme)
 		break;
 	case NL80211_AUTHTYPE_NETWORK_EAP:
 		WL_DBG(("network eap\n"));
-		// fallthrough
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 0)
+		fallthrough;
+#endif
 	default:
 		val = 2;
 		WL_ERR(("invalid auth type (%d)\n", sme->auth_type));

